README.md

```diff
@@ -332,7 +336,7 @@ For the latest troubleshooting information please read both open and closed tick
 
 <https://github.com/digininja/DVWA/issues>
 
-Before submitting a ticket, please make sure you are running the latest version of the code from the repo. This is not the latest release, this is the latest code from the master branch.
+Before submitting a ticket, please make sure you are running the latest version of the code from the repo. This is the latest release, this is the latest code from the master branch.
 
```

Comment:
The "not" should be there, it is telling them not to use the latest release, to use the code from the master branch.

Reply:
Oops thanks for the explanation! Just misunderstood the intention.  Let me revert this change and also leave a comment in the Chinese translation.

Reply:
Done.